src/toolbars/MeterToolBar.cpp

```diff
@@ -146,43 +144,87 @@ void MeterToolBar::ReCreateButtons()
 void MeterToolBar::Populate()
 {
    SetBackgroundColour( theTheme.Colour( clrMedium  ) );
-   Add((mSizer = safenew wxGridBagSizer()), 1, wxEXPAND);
 
    if( mWhichMeters & kWithRecordMeter ){
       //JKC: Record on left, playback on right.  Left to right flow
       //(maybe we should do it differently for Arabic language :-)  )
+      mRecordSetupButton = safenew AButton(this);
+      mRecordSetupButton->SetLabel({});
+      mRecordSetupButton->SetName(_("Record"));
+      mRecordSetupButton->SetImages(
+         theTheme.Image(bmpUpButtonSmall),
+         theTheme.Image(bmpHiliteUpButtonSmall),
+         theTheme.Image(bmpDownButtonSmall),
+         theTheme.Image(bmpHiliteButtonSmall),
+         theTheme.Image(bmpUpButtonSmall));
+      mRecordSetupButton->SetIcon(theTheme.Image(bmpMic));
+      mRecordSetupButton->SetButtonType(AButton::Type::FrameButton);
+      mRecordSetupButton->SetMinSize({toolbarSingle, toolbarSingle});
+      mRecordSetupButton->Bind(wxEVT_BUTTON, [this](wxCommandEvent&)
+      {
+         mRecordMeter->ShowMenu(
+            mRecordMeter->ScreenToClient(
+               ClientToScreen(mRecordSetupButton->GetClientRect().GetRightBottom())
+            )
+         );
+      });
+
+
       mRecordMeter = safenew MeterPanel( &mProject,
                                 this,
                                 wxID_ANY,
                                 true,
                                 wxDefaultPosition,
-                                wxSize( 260, 28 ) );
+                                wxSize( 260, toolbarSingle) );
       /* i18n-hint: (noun) The meter that shows the loudness of the audio being recorded.*/
       mRecordMeter->SetName( XO("Record Meter"));
       /* i18n-hint: (noun) The meter that shows the loudness of the audio being recorded.
        This is the name used in screen reader software, where having 'Meter' first
        apparently is helpful to partially sighted people.  */
       mRecordMeter->SetLabel( XO("Meter-Record") );
-      mSizer->Add( mRecordMeter, wxGBPosition( 0, 0 ), wxDefaultSpan, wxEXPAND );
    }
 
    if( mWhichMeters & kWithPlayMeter ){
```

Comment:
Does mWhichMeters still need to be a bitwise or of non-exclusive choices?

There was old, old code for an alternative combined play & record meter toolbar in one, instead of the separate ones.  I forget when that was removed, but it appears the program is still constructing this third, combined meter toolbar, and yet never showing it and giving you no user interface to turn it on!